platform/posix/CMakeLists.txt

```diff
@@ -9,6 +9,13 @@ target_include_directories( clock_posix
                               PRIVATE
                                 ${PLATFORM_DIR}/include )
 
+# Install clock abstraction as library.
+if(INSTALL_PLATFORM)
+    install(TARGETS
```

Comment:
Shouldn't the include path destination be specified as well?

Reply:
Yes, it is possible to also install the headers in this file. However, what ends up happening is that the `make install` output will no longer group libraries and headers together. To fix that, the headers are installed here instead:
https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1519/files/96807066b1b35fd543d45f6a57063325cc28dd69#diff-7021d478f66d585e3f9025b7212b1ca40b1db63995592041e28b5dd819c06e53R154-R157
Actually, I did want to also install all libraries in this file but it is not backwards compatible with older versions of CMake.

Reply:
>  install all libraries in this file but it is not backwards compatible with older versions of CMake.
What aspect about installing all libraries in the file is not backwards compatible with older version of CMake?

Reply:
The cmake file that calls `add_library()` for a target is also where `install()` needs to be called for that target in older versions of cmake.